src/Test.UnitTests.Sarif.Multitool.Library/ExportValidationRulesDocumentationCommandTests.cs

```diff
@@ -36,5 +45,31 @@ private class TestRule : SarifValidationSkimmerBase
                 { "TEST0001_TEST_Note_Default", new MultiformatMessageString{ Text="default text"} }
             };
         }
+
+        private class TestRule2 : SarifValidationSkimmerBase
+        {
+            public override string Id => "TEST0002";
+            public override string Name => "TEST2";
+            public override FailureLevel DefaultLevel => FailureLevel.Note;
+            public override MultiformatMessageString FullDescription => new MultiformatMessageString { Text = null };
```

Comment:
>ext = null [](start = 104, length = 10)

`null` isn't a valid value for `Text` (although the SDK incorrectly accepts it). Just use `FullDescription => null;`,

Reply:
(similarly both the `ShortDescription` and the `FullDescription` in `TestRule3`.)

---
In reply to: [493032568](https://github.com/microsoft/sarif-sdk/pull/2082#discussion_r493032568) [](ancestors = 493032568)